local-kubernetes/README.md

```diff
@@ -10,7 +10,13 @@ Create the buying catalogue namespace - `kubectl apply -f local-namespace.yml`
 
 To run the system, you need to create a secret in kubernetes to access the private container registry, as per the [Connect Local Kubernetes with our Private Container Registry Instructions](../Docs/DevSetup/k8s-private-registry.md).
 
-You will also need a secret for the local instances of sql server - `kubectl create secret generic mssql --from-literal=SA_PASSWORD="<YOUR PASSWORD>" -n buyingcatalogue`. **Password must be follow this [policy](https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver15#password-complexity)**.
+You will also need secrets for the local instances of sql server and identity server 
+
+### mssql server:
+- `kubectl create secret generic mssql --from-literal=SA_PASSWORD="<YOUR PASSWORD>" -n buyingcatalogue`. **Password must be follow this [policy](https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver15#password-complexity)**.
+
+### identity server:
+- `kubectl create secret generic oidc-client-secret --from-literal=OIDC_CLIENT_SECRET="<YOUR CLIENT SECRET>" -n buyingcatalogue`
```

Comment:
does this client secret need to be something in particular to work with the front end?

Reply:
The front-end and back-end are configured to both use the same secret, I didn't assume it needs to be something specific as that would defeat the purpose of a secret. I went ahead and tested changing this secret to some arbitrary string and - to my surprise it did not work, however it did not work because this env. variable injected by k8 does not override the 'clients': { ..., 'secret': 'value' } in Isapi's appconfig.json which looks like an issue with Isapi. When Isapi is taken outside of the cluster and the value in appsettings is set to match the kube secret everything is fine. I'll try to get Isapi to behave accordingly (change the secret's value to come from env. var if present) and that will fix this.

Reply:
Update: 
Turns out I glazed over the fact that isapi's appsettings has an **array** of clients, therefore the env variable set in the yaml manifest was't overriding anything.